src/test/java/JsonParserTests.java

```diff
@@ -0,0 +1,131 @@
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import parser.JsonParser;
+import parser.Parser;
+import shop.Cart;
+
+
+import java.io.File;
+
+
+public class JsonParserTests {
+
+
+    private Parser jsonParser;
+    private Cart cart;
+
+
+    @BeforeEach
+    void setUp() {
+        jsonParser = new JsonParser();
+    }
+
+    @AfterEach
+    void tearDown() {
+        String pathName = "src/main/resources/";
+
+        File file = new File(pathName);
```

Comment:
Take a look on https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html
I recommend to use it

Reply:
file = new File("src/main/resources/");
        for (File item : file.listFiles()) {
            if (!(item.getName().equals("andrew-cart.json") | item.getName().equals("eugen-cart.json"))) {
                FileUtils.deleteQuietly(item);
            }